unitylibs/core/workflow/workflow-acrobat/target-config.json

```diff
@@ -24,6 +24,7 @@
     "nonpdfSfuProductScreen": ["word-to-pdf", "jpg-to-pdf", "ppt-to-pdf", "excel-to-pdf", "png-to-pdf", "createpdf", "chat-pdf", "chat-pdf-student", "summarize-pdf"],
     "mfuUploadAllowed": ["combine-pdf", "rotate-pages", "chat-pdf", "chat-pdf-student", "summarize-pdf"],
     "mfuUploadOnlyPdfAllowed": ["combine-pdf"],
+    "experimentationOn": ["add-comment"],
```

Comment:
I think this is fine for now, but may want to identify an alternative approach to this for reduced code maintainability.